server/compute/diffexp_cxg.py

```diff
@@ -87,24 +87,11 @@ def diffexp_ttest(adaptor, maskA, maskB, top_n=8, diffexp_lfc_cutoff=0.01):
 
     if is_sparse:
         for cols in col_partitions:
-            futures.append(
-                executor.submit(
-                    # TODO: remove this temporary workaround when TileDB fixes the issue with multi_index
-                    _mean_var_sparse_ab, adaptor.open_array("X"), row_selector_A, nA, row_selector_B, nB, cols
-                )
-            )
+            futures.append(executor.submit(_mean_var_sparse_ab, matrix, row_selector_A, nA, row_selector_B, nB, cols))
```

Comment:
Undoes this:
* https://github.com/chanzuckerberg/single-cell-explorer/pull/60/files